packages/@aws-cdk/aws-elasticloadbalancing/lib/load-balancer.ts

```diff
@@ -248,23 +248,25 @@ export class LoadBalancer extends Resource implements IConnectable {
   private readonly elb: CfnLoadBalancer;
   private readonly securityGroup: SecurityGroup;
   private readonly listeners: CfnLoadBalancer.ListenersProperty[] = [];
+  private readonly instances: IInstance[] = [];
```

Comment:
This holds on to `IInstances`, for the sole purpose of using them as `IConnectable`s. This collection doesn't need to be as specific as it is. If we just need "anything that's `IConnectable`" we can just write that. By writing `IInstance` we are excluding other things that could be connectable.

But I'm now noticing that `ILoadBalancerTarget extends IConnectable`, so we *already have* a list of `IConnectable`s... it's `this.targets` !